### ISSUE TITLE
Move README to Sphinx docs (and ReadTheDocs)

### ISSUE THREAD

**Issue by IanLee1521** [mej/nhc#3]
Michael —

Congratulations on the move to GitHub! I understand this might not really have been [your first choice](http://mej0.blogspot.com/2012/12/dragged-onto-gitwagon.html), but for what it’s worth, I’m happy about it. :)

I wanted to reach out and inquire whether you had any thoughts about "dissecting" the README.md file and moving that into more of a true documentation guide? I would be happy to do some work and submit a pull request for you. My thought would be to use [Sphinx](http://sphinx-doc.org/) and [ReadTheDocs.org](https://readthedocs.org) to build and maintain a live copy of the documentation. Again, would like to get some input before diving into the work.

Thoughts?

**Comment by mej** [comment]
Yes, I was skeptical about Git at first, but I've totally converted just about everything I do.  The only issue is the one I called out in my journal post -- binaries in repositories.  While I don't use it for my RPM packaging work, in all other respects, I'm a total convert.  I've even been using `git-svn` to work with all Subversion repositories, and have since pretty much the day I wrote that post.  :-)

As for your request, I'm all for it!  The truth is I've been wondering how to go about doing that, and I just haven't had time yet to investigate how to get documentation into ReadTheDocs.org or any other system.  If you have time and willingness, I'd love the help!  Thanks!!  :-)

**Comment by IanLee1521** [comment]
Sounds good. I'm quite a fan myself. We can continue this offline (my username @gmail.com if you want), but I've actually had reasonable success with using GIT for my building RPMS (about 50 spec / source pairs for various tools).

Back on this topic, do you have any preference for my working off either the master or dev branch? I noticed just noticed that there were both, and I should probably work off dev, but the last PR #2 was onto master, so... Just looking for your preference.

I'm starting the work to move to Sphinx (which is what RTD uses) over on [IanLee1521/nhc @ sphinx-docs](https://github.com/IanLee1521/nhc/tree/sphinx-docs). I also created a [ReadTheDocs.org project](https://readthedocs.org/projects/lbnl-node-health-check/) and added you as a maintainer (I can step down after getting it up and running). I'm temporarily pointing it to my branch, just to make sure things are building fine, once this branch / fix gets merged we can swing RTD back to this repo instead.

**Comment by IanLee1521** [comment]
Ok, I've gotten the README moved over, headings converted to ReST vs MD formatted, and I think I got all of the code blocks. I'll pick up with tables tomorrow. Then will probably look to you, @mej, a bit to help steer how it might get broken up into separate pages. Off the top of my head, I'm thinking `Getting Started`, `Configuration`, and `Customization` will be the three major pieces, and the easiest to split.

**Comment by mej** [comment]
I'm at SC15 this week, so things are a bit hectic at the moment, but I'll look at this as soon as I can!  :-)
